### ISSUE TITLE
Add freeze(obj) and unfreeze(obj) methods

### ISSUE THREAD

**Issue by misko321** [gsklee/ngStorage#178]
The **ngStorage** by default watches for any changes of an added object, to update the localStorage. In some cases however (like this one: https://stackoverflow.com/questions/32856511/localstorage-value-got-change-with-fabric-js/32966097#32966097) it would be very helpful to disable the automatic updates by some sort of *freeze(obj)* method which would prevent the object from further modifications. Optionally, *unfreeze(obj)* method in turn would reenable the watch, and start syncing the object and its localStorage representation again.

**Comment by btm1** [comment]
+1 yes plz

**Comment by rhclayto** [comment]
Yes, also a +1 from me. This would greatly expand the flexibility of this essential module.

**Comment by TomekSzymanski** [comment]
+1

**Comment by jleszczynski** [comment]
+1
